setup.py

```diff
@@ -1,4 +1,5 @@
 from setuptools import setup
+import multiprocessing
```

Comment:
On Sep 20, 2015, at 4:20 PM, Michael Silver wrote:
> In setup.py:
> 
> > @@ -1,4 +1,5 @@
> >  from setuptools import setup
> > +import multiprocessing
> 
> It's there because nose requires it. Otherwise I get a disgusting error: [snip]

If the dependency is that obscure, maybe it could use a comment?  Something like "import to prevent nose error: 
TypeError: 'NoneType' object is not callable"?

-Sue